salt/hg/init.sls

```diff
@@ -171,6 +171,10 @@ apache2:
       - file: /etc/apache2/mods-enabled/*
```

Comment:
We probably want to add `/etc/apache2/conf-enabled/*` to this watch so that configuration changes in that directory reloads apache automatically!

Reply:
Roger that 🫡